test/gtest/tuna_net.cpp

```diff
@@ -101,26 +131,27 @@ void TestSolverPredictionModel(miopen::conv::ProblemDescription& problem,
 #else
     std::ignore = problem;
     std::ignore = expected_solver;
+    std::ignore = device_architecture
```

Comment:
🐛 Missing semicolon. I suspect this wasn't built with MIOPEN_ENABLE_AI_IMMED_MODE_FALLBACK=Off

Reply:
Fixed in <a href=https://github.com/ROCm/MIOpen/pull/2785/commits/6bee000efac77beff0bd2d55c7c88af0c7a9cccc>PR 2785, commit 6bee000ef</a>